modules/ROOT/pages/index.adoc

```diff
@@ -25,6 +25,44 @@ Installing Runtime Fabric within an existing Kubernetes-based PaaS is not suppor
 
 If you are already using a PaaS solution, MuleSoft recommends deploying Runtime Fabric in parallel with your PaaS. This enables you to take advantage of the complete benefits of Anypoint Platform.
 
+== Shared Responsibility for On-Premises Runtime Fabric
+
+The successful operation of on-premises Runtime Fabric instances is a shared responsibility. It is critical to understand which areas you must manage and which areas are managed by MuleSoft. Without the required infrastructure and proper maintenance, Runtime Fabric will not operate successfully.
+
+The following image illustrates MuleSoft and customer responsibilities for on-premises Runtime Fabric instances:
+
+image::sharedresponsibility.png[Runtime Fabric Shared Responsiblities]
+
+* MuleSoft responsibility:
++
+MuleSoft manages the Runtime Fabric control plane and is responsible for the delivered components, Runtime Fabric appliance, Runtime Fabric agent, Mule runtime engine, and other dependencies for Mule appications.
+
+* Customer responsibility:
++
+Customers are responsible for provisioning, configuring, and managing the infrastructure required for Runtime Fabric and Runtime Fabric clusters.  
```

Comment:
Runtime Fabric and Runtime Fabric clusters is redundant. Suggest removing "Runtime Fabric clusters"

Reply:
Updated this - thank you!